### PR TITLE
haskell-modules: update tensorflow for 1.14

### PR DESCRIPTION
**Pull request by yorickvP**
I've updated the tensorflow packages so they work with tensorflow 1.14.
Please let me know if there is a better way to put this in nixpkgs.

###### Notify maintainers

cc @basvandijk @peti

### CODE REVIEW

File: pkgs/development/haskell-modules/tensorflow-packages.nix
Comment (cdepillabout) [inline pkgs/development/haskell-modules/tensorflow-packages.nix]: It would be great if there was a separate updater script that basically did the above.

A bunch of other places in nixpkgs have an actual updater script that bumps versions and re-runs commands like this.

Having a script like this would make it much easier for other maintainers to keep the tensorflow stuff updated.

**Comment by peti** [comment]
Where do the packages listed in `pkgs/development/haskell-modules/tensorflow-packages.nix` come from? Are those packages that are on Hackage?

File: pkgs/development/haskell-modules/tensorflow-packages.nix
Comment (cdepillabout) [inline pkgs/development/haskell-modules/tensorflow-packages.nix]: I was thinking that these tensorflow packages could probably go in the `non-hackage-packages.nix` file (since that is what they are), but if you also add an update script, it might just be easier to keep them in a separate file.

**Comment by cdepillabout** [comment]
@GrahamcOfBorg build haskellPackages.tensorflow haskellPackages.tensorflow-core-ops  haskellPackages.tensorflow-logging  haskellPackages.tensorflow-opgen haskellPackages.tensorflow-ops haskellPackages.tensorflow-proto haskellPackages.tensorflow-records haskellPackages.tensorflow-records-conduit haskellPackages.tensorflow-test

**Review by peti — CHANGES_REQUESTED** [review]
Why does this PR package non-Hackage versions of those packages? 

Shouldn't we distribute released versions?

**Comment by cdepillabout** [comment]
@peti It looks like the tensorflow libraries don't build with the latest `proto-lens-protoc`.  yorickvP has apparently fixed them up and sent a PR upstream:

https://github.com/tensorflow/haskell/pull/247

I have confirmed that all the packages from this PR build correctly, but I imagine it might just be easier for us to wait until upstream merges in the above PR and makes a new release.  When that happens we will be able to get rid of the stuff from this PR.

**Comment by yorickvP** [comment]
I've asked upstream to do a release

**Pull request closed without merging**
